examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -37,7 +37,7 @@
 
 
 fig, ax = plt.subplots(figsize=(12, 4))
-average_week_demand = df.groupby(["weekday", "hour"]).mean()["count"]
+average_week_demand = df.groupby(["weekday", "hour"]).mean(numeric_only=True)["count"]
```

Comment:
What do you think of specifying `"count"` first before the mean?

```suggestion
average_week_demand = df.groupby(["weekday", "hour"])["count"].mean()
```

Reply:
Seems like it works.  Might make a little more sense too if I am interpreting things right.